abixen-platform-core/src/main/java/com/abixen/platform/core/model/impl/Comment.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2010-present Abixen Systems. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.abixen.platform.core.model.impl;
+
+import com.abixen.platform.core.model.CommentBase;
+import com.abixen.platform.core.model.web.CommentWeb;
+
+import javax.persistence.*;
+
+
+@Entity
+@Table(name = "comment")
+@SequenceGenerator(sequenceName = "comment_seq", name = "comment_seq", allocationSize = 1)
+public class Comment extends AuditingModel implements CommentBase<Layout>, CommentWeb {
```

Comment:
I think should be replaced by Comment class, because of Comment parent field which is missing for now.